schema/shacl/SCHEMA_QUDT_NoOWL-v2.1.ttl

```diff
@@ -3566,6 +3571,59 @@ voag:supersededBy
   vaem:namespace "http://www.linkedmodel.org/schema/dtype#"^^xsd:anyURI ;
   vaem:namespacePrefix "dtype" ;
 .
+dtype:EnumeratedValue
+  a rdfs:Class ;
+  a sh:NodeShape ;
+  dcterms:description "This is a placeholder class that is used to represent the values of an enumeration. It is not intended to be used directly." ;
+  rdfs:isDefinedBy <http://www.linkedmodel.org/schema/dtype> ;
+  rdfs:label "DTYPE Enumerated Value" ;
+  rdfs:subClassOf rdfs:Resource ;
+.
+dtype:Enumeration
+  a rdfs:Class ;
+  a sh:NodeShape ;
+  dcterms:description "A placeholder." ;
+  rdfs:isDefinedBy <http://www.linkedmodel.org/schema/dtype> ;
+  rdfs:label "DTYPE Enumeration" ;
+  rdfs:subClassOf rdfs:Resource ;
+.
+dtype:code
+  a rdf:Property ;
+  vaem:comment "A property for expressing an encoded value. The range has been set to \"xsd:anySimpleType\" to allow for a variety of scalar datatypes." ;
+  rdfs:isDefinedBy <http://www.linkedmodel.org/1.1/schema/dtype> ;
+  rdfs:label "code" ;
+.
+dtype:defaultValue
+  a rdf:Property ;
+  rdfs:isDefinedBy <http://www.linkedmodel.org/schema/dtype> ;
+  rdfs:label "default value" ;
+.
+dtype:literal
+  a rdf:Property ;
+  rdfs:isDefinedBy <http://www.linkedmodel.org/schema/dtype> ;
+  rdfs:label "literal" ;
+.
+dtype:numericUnion
+  a rdfs:Class ;
+  a rdfs:Datatype ;
+  a sh:NodeShape ;
+  dcterms:description "This is a placeholder for a datatype that is the union of a set of numeric datatypes. The union of numeric datatypes is not a datatype, but a class of datatypes." ;
+  rdfs:isDefinedBy <http://www.linkedmodel.org/schema/dtype> ;
+  rdfs:label "numeric union" ;
+  rdfs:subClassOf rdfs:Resource ;
+.
+dtype:order
+  a rdf:Property ;
+  dcterms:description "The property \"dtype:order\" provides a means to specify a precedence. One use of order is in specifying ordered enumerations such as \"voag:ConfidentialityLevel\".  A similar property, but with an important type difference, is \"vaem:orderIndex\". This is for use on property occurrences in class axioms where it can be placed on a restriction to specify how that property may be transformed into other representations where ordering has some importance, for example, in XML Schema sequences.  Whereas \"vaem:order\" is a datatype property, \"vaem:orderIndex\" is an annotation property." ;
+  rdfs:comment "The \"order\" property specifies a value that can establish precedence and ordinality. One use of order is in make an enumeration an ordinal type. For example \"RiskLevel\" might have the values \"very low\", \"low\", \"medium\", \"high\" and \"very high\". A query that asks for all risks that are greater than \"medium\" would use the ordinal values and return those resources that have “medium”, “high” and “very high” ordinality." ;
+  rdfs:isDefinedBy <http://www.linkedmodel.org/schema/dtype> ;
+  rdfs:label "order" ;
+.
+dtype:value
+  a rdf:Property ;
+  rdfs:isDefinedBy <http://www.linkedmodel.org/schema/dtype> ;
+  rdfs:label "value" ;
+.
```

Comment:
I have a concern about expressing all the details of these non-imported resources. Can we discuss in the afternoon?